### PR TITLE
added fourier_reverse! operations

### PR DESCRIPTION
**Pull request by RainerHeintzmann**
Interestingly the problem was more complex than I thought. 
One could also add a reverse_by_ft() method, which is easy via ft() but maybe not so easy via fft()?

### CODE REVIEW

File: src/utils.jl [hunk 1]
```diff
@@ -397,3 +399,60 @@ Base.@propagate_inbounds function Base.setindex!(A::PaddedView{T, N}, v, i::Vara
     return A
 end
  =#
+
+ """
+    odd_view(arr)
+
+creates a view of `arr` the for each even dimension excludes the
```
Comment (roflmaostc) [inline src/utils.jl]: grammer: the for each

File: src/utils.jl [hunk 2]
```diff
@@ -1,6 +1,8 @@
 export rft_size, fft_center, fftpos
 export expanddims, fourierspace_pixelsize, realspace_pixelsize
 export δ
+export odd_view, fourier_reverse!
```
Comment (roflmaostc) [inline src/utils.jl]: Would be against exporting `odd_view`. Is not that important and might pollute namespaces.

**Review by roflmaostc — COMMENTED** [review]
Add documentation for both functions to the web docs too

File: src/utils.jl (hunk 2 above)
Comment (RainerHeintzmann) [reply src/utils.jl]: done

File: src/utils.jl (hunk 1 above)
Comment (RainerHeintzmann) [reply src/utils.jl]: done

Comment (RainerHeintzmann) [reply src/utils.jl]: also added docs

**Pull request merged**